src/webui/api/appcontroller.cpp

```diff
@@ -119,6 +119,8 @@ void AppController::preferencesAction()
     data[u"file_log_delete_old"_s] = app()->isFileLoggerDeleteOld();
     data[u"file_log_age"_s] = app()->fileLoggerAge();
     data[u"file_log_age_type"_s] = app()->fileLoggerAgeType();
+    // Delete torrent files on torrent removal
+    data[u"delete_torrent_files_as_default"_s] = pref->deleteTorrentFilesAsDefault();
```

Comment:
I see that u just follow existing wording but isn't "by default" is more correct/usual?

Reply:
So `delete_torrent_contents_default` or `delete_torrent_contents_by_default`?

Reply:
> delete_torrent_content_files_by_default

This. But...
IMO, we can avoid such extra "by default" in (almost) all global options names since they refer to default values "by default".

Reply:
OK, all done.